### PR TITLE
Add dibels to filters table

### PR DESCRIPTION
**Pull request by GabyRangelB**
# Pull Request

## Summary & Motivation

[//]: # (When merged, this pull request will...)

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

    _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

## Troubleshooting
- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206154388954840

### CODE REVIEW

**Review by anthonygwalters — DISMISSED** [review]
(discussed in o3)

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql [hunk 1]
```diff
@@ -75,6 +75,31 @@ with
...
         where rn_subj_round = 1 and test_round = 'EOY'
+    ),
+
+    composite_only as (
+        select
+            cast(left(bss.school_year, 4) as int) as academic_year,
+            bss.student_primary_id as student_number,
+            'DIBELS Benchmark' as assessment_type,
+            bss.benchmark_period as mclass_period,
+            u.measure as mclass_measure,
```
Comment (cbini) [inline src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql:86]: looks like you can remove `assessment_type ` and `mclass_measure `

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql [hunk 2]
```diff
@@ -75,6 +75,31 @@ with
...
+            u.level as mclass_measure_level,
+        from {{ ref("stg_amplify__benchmark_student_summary") }} as bss
+        inner join
+            {{ ref("int_amplify__benchmark_student_summary_unpivot") }} as u
+            on bss.surrogate_key = u.surrogate_key
+        where
+            cast(left(bss.school_year, 4) as int) = {{ var("current_academic_year") }}
```
[the remark this answers is not in the document]
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql]: Done!

What is the best way to know when you've made improvements to the tables? academic_year was not a field that existed when this code was created by you, back in August for DIBELS (where I copied this code from).

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql [hunk 3]
```diff
@@ -75,6 +75,31 @@ with
             end as iready_proficiency,
         from {{ ref("base_iready__diagnostic_results") }}
         where rn_subj_round = 1 and test_round = 'EOY'
+    ),
+
+    composite_only as (
+        select
+            cast(left(bss.school_year, 4) as int) as academic_year,
```
[the remark this answers is not in the document]
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql]: Done!

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql (hunk 1 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql]: True, true. Done!

**Review by cbini — APPROVED** [review]

**Pull request merged**